### PR TITLE
NEXUS-20839 Helm plugin should be disabled when HA-C is enabled

### PR DESCRIPTION
**Pull request by mlukaretkyi**
Jira: https://issues.sonatype.org/browse/NEXUS-20839

### CODE REVIEW

File: nexus-repository-helm/src/test/java/org/sonatype/repository/helm/internal/HelmRecipeTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class HelmRecipeTest
+    extends TestSupport
+{
+  private static final String FORMAT_NAME = HelmFormat.NAME;
```
Comment (chernikovmaksym) [inline nexus-repository-helm/src/test/java/org/sonatype/repository/helm/internal/HelmRecipeTest.java]: I don't think variable is needed, just use static import

File: nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedRecipe.groovy [hunk 2]
```diff
@@ -89,6 +89,7 @@ class HelmHostedRecipe
 
     [indexMatcher(), packageMatcher()].each { matcher ->
       builder.route(new Route.Builder().matcher(matcher)
+          .handler(formatHighAvailabilitySupportHandler)
```
Comment (sergeyonofreychuck) [inline nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedRecipe.groovy]: move HA handler after security handler (in all routes)

Comment (mlukaretkyi) [reply nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedRecipe.groovy]: fixed, please take a look 
https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/72/commits/ce51f279a8bb6093fc738d5dc0bb650caf6be079

File: nexus-repository-helm/src/test/java/org/sonatype/repository/helm/internal/HelmRecipeTest.java (hunk 1 above)
Comment (mlukaretkyi) [reply nexus-repository-helm/src/test/java/org/sonatype/repository/helm/internal/HelmRecipeTest.java]: fixed, please take a look
https://github.com/sonatype-nexus-community/nexus-repository-helm/pull/72/commits/ce51f279a8bb6093fc738d5dc0bb650caf6be079

**Pull request merged**